### ISSUE TITLE
[UI] Initial Data Plane Dashboard

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#3697]
#### Current Behavior
The Dashboard shows only service mesh control plane components

#### Desired Behavior
The Dashboard should display service mesh data plane components (proxies) as well.

#### Implementation
Proxies details should be displayed through GraphQL subscription.

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

**Comment by Abhishek-kumar09** [comment]
Is the design ready for data plane dashboard? @leecalcote

**Comment by leecalcote** [comment]
@Abhishek-kumar09 yes, it's just adding another row to the existing control plane dashboard with details of the proxies deployed in the system. // @bariqhibat
